src/main/java/at/hannibal2/skyhanni/utils/renderables/Renderable.kt

```diff
@@ -1609,34 +1476,9 @@ interface Renderable {
                 GuiRenderUtils.drawTexturedRect(0, 0, width, height, uMin, uMax, vMin, vMax)
                 GlStateManager.color(1f, 1f, 1f, 1f)
 
-                GlStateManager.translate(padding.toFloat(), padding.toFloat(), 0f)
+                DrawContextUtils.translate(padding.toFloat(), padding.toFloat(), 0f)
                 input.render(posX + padding, posY + padding)
-                GlStateManager.translate(-padding.toFloat(), -padding.toFloat(), 0f)
-            }
-        }
-
-        fun image(
-            texture: ResourceLocation,
```

Comment:
was this unused?

Reply:
yes, see https://github.com/hannibal002/SkyHanni/pull/3890#discussion_r2061261006